Fitness/Views/ProBioPageViewController.swift

```diff
@@ -0,0 +1,313 @@
+        flowLayout.minimumLineSpacing = 0
+        flowLayout.scrollDirection = .vertical
+        
+        linksTitleLabel = UILabel()
+        linksTitleLabel.text = "FOLLOW ME"
+        linksTitleLabel.font = ._12MontserratBold
+        linksTitleLabel.textColor = .black
+        linksTitleLabel.textAlignment = .center
+        linksTitleLabel.sizeToFit()
+        scrollView.addSubview(linksTitleLabel)
+        
+        linksCollectionView = UICollectionView(frame: .zero, collectionViewLayout: flowLayout)
+        linksCollectionView.delegate = self
+        linksCollectionView.dataSource = self
+        linksCollectionView.backgroundColor = .white
+        linksCollectionView.register(PersonalLinkCollectionViewCell.self, forCellWithReuseIdentifier: PersonalLinkCollectionViewCell.identifier)
+        scrollView.addSubview(linksCollectionView)
+        
+        backButton = UIButton()
+        backButton.setImage(UIImage(named: "back-arrow"), for: .normal)
+        backButton.sizeToFit()
+        backButton.addTarget(self, action: #selector(self.back), for: .touchUpInside)
+        view.addSubview(backButton)
+        view.bringSubviewToFront(backButton)
+        
+        setPro()
+        setupConstraints()
+    }
+    
+    @objc func back() {
+        navigationController?.popViewController(animated: true)
+    }
+    
+    func setPro() {
+        nameLabel.text = pro.name.uppercased()
+        
+        let style = NSMutableParagraphStyle()
+        style.lineSpacing = 5
+        let attributes = [NSAttributedString.Key.paragraphStyle : style, NSAttributedString.Key.font : UIFont._14MontserratLight]
+        bioTextView.attributedText = NSAttributedString(string: pro.bio, attributes: attributes)
+        bioTextView.textAlignment = .center
+        
+        bioTitleLabel.text = pro.summary
+        
+        expertiseTextView.text = pro.expertise.joined(separator: " · ")
+        
+        linksCollectionView.reloadData()
+        
+        routinesView = AllProRoutinesView(routines: pro.routines)
+        scrollView.addSubview(routinesView)
+        
+        profilePicView.image = pro.profilePic
+    }
+    
+    func setupConstraints() {
+        let backButtonHeight = 19
+        let backButtonLeading = 20
+        let backButtonTop = 30
+        let backButtonWidth = 23
+        let linksTitleCollectionViewSpacing = 12
+        let profilePicHeightWidthRatio = 1.16
+       
+        backButton.snp.makeConstraints { make in
+            make.leading.equalTo(view).offset(backButtonLeading)
+            make.top.equalTo(view.safeAreaLayoutGuide.snp.top).offset(backButtonTop)
+            make.width.equalTo(backButtonWidth)
+            make.height.equalTo(backButtonHeight)
+        }
+        
+        scrollView.snp.makeConstraints{ make in
+            make.top.leading.trailing.bottom.equalToSuperview()
+            make.width.equalTo(view.snp.width)
```

Comment:
isn't his also equal to the superView then?

Reply:
Yes, good catch